input/blog/2018-07-06-cake-v0.29.0-released.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cake v0.29.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.29.0 of Cake has been released.
+
+We're really excited by this release which has some splendid features and improvements!
+
+In this blog post we'll highlight the new `exclusive` parameter which adds support for executing a single task without dependencies. 
+You can find a full list of changes at the end of this post.
+
+Contributions were included from:
+
+- [devlead](https://github.com/devlead)
+- [gep13](https://github.com/gep13)
+- [patriksvensson](https://github.com/patriksvensson)
+- [grdylan](https://github.com/grdylan)
+- [paulomorgado](https://github.com/paulomorgado)
+- [mholo65](https://github.com/mholo65)
+- Kevin LaBranche
+- [rh072005](https://github.com/rh072005)
+- [louisfischer](https://github.com/louisfischer)
+- [pavzaj](https://github.com/pavzaj)
+- Michael Samorokov
+- [daveaglick](https://github.com/daveaglick)
+- NEXOGEN\kemenya
+
+<!--excerpt-->
+
+### Exclusive
+
+When you've executed a series of tasks and it fails on the last one, wouldn't it be nice if you could execute a single task, without requiring all of its dependencies to execute again? Well you're in luck that just got added into 0.29.0, beyond retrying failed operations - this can really speed things up while developing and debugging a script.
```

Comment:
I think we need a slight modification to this section...

> Well you're in luck, this just got added into 0.29.0!  Beyond retrying failed operations - this can really speed things up while developing and debugging a script.

Reply:
Any suggestion or just remove?

Reply:
@devlead that quoted text above is my suggestion 😄 Just separated the sentence, and added a comma.